content/early-years-teaching-training.md

```diff
@@ -0,0 +1,97 @@
+      - "Graduate early years teacher training"     
+  EY-graduate-employment:
+    name: feature_table
+    arguments:
+      - Cost: "£7000"
+        Salary: "Provided by employer"
+        Funding available: "Yes"
+        Length of training: "Typically 1 academic year"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate employment-based early years teacher training"
+  EY-self-funded:
+    name: feature_table
+    arguments:
+      - Cost: "Self-funded"
+        Salary: "No"
+        Funding available: "No"
+        Length of training: "3 months"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate Assessment-Only training"
+---
+
+## Becoming an early years teacher
+ 
+As an early years teacher, you will work with young children from birth up to age five. Your input will have a lasting and positive effect on children’s development and wellbeing.
+ 
+By training to deliver the Early Years Foundation Stage, you’ll learn how to use your knowledge and skills to help young children enjoy high standards of teaching. These early years are critical for young children’s development, and you’ll be opening their minds to new ideas every day.
+ 
+###How to become an early years teacher
+ 
+To qualify, you’ll need Early Years Teacher Status, which is different from Qualified Teacher Status (the equivalent for teaching children aged three to 18). Your training will enable you to meet the [Teachers’ Standards (Early Years)](https://www.gov.uk/government/publications/early-years-teachers-standards).
+ 
+To start early years teacher training, you’ll need these qualifications (or be able to deomnstrate an equivalent standard):
+
+* Maths grade 4 (C)
+* English grade 4 (C)
+* Science grade 4 (C)
+
+Graduate training also requires a Bachelor’s degree with honours, class 2:2 or higher.
+ 
+To apply for your early years teacher training place, please contact one of our [training providers](https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations).
+
+## Ways to train
+
+$EY-undergraduate$
+
+A full-time bachelor’s degree leading to EYTS for those studying for a degree in an early childhood-related subject. Tuition fee loans are available from [Student Finance England
+(SFE)](https://www.gov.uk/student-finance).
+
+$EY-graduate$
+ 
+There are also bursaries available of up to £5000 for graduates with a first class degree, £4000 for graduates with a 2:1, and £2000 for graduates with a 2:2.
+ 
+$EY-graduate-employment$
+
+This one-year part-time route is for graduates working in an early years setting who need further training to meet the Teachers’ Standards (Early Years). Funding of up to £14,000 is available. This would cover course fees of up to £7000, plus £7000 for your employer, to help cover their extra costs while you train.
```

Comment:
```suggestion
This one-year part-time route is for graduates working in an early years setting who need further training to meet the Teachers’ Standards (Early Years). Funding of up to £14,000 is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train.
```